### ISSUE TITLE
performance about the model

### ISSUE THREAD

**Issue by guoxiaolu** [flyyufelix/cnn_finetune#23]
Awesome job and it is truly convenient. I have used the resnet101 model based on tensorflow, and evaluated it on ILSVRC2015 validation set, however, the top-1 error is about 35% and higher more than 10% than He's given result (https://github.com/KaimingHe/deep-residual-networks). So can you give the performance result of these models?
Thank you very much~
@flyyufelix

**Comment by flyyufelix** [comment]
Thanks for providing the test results. I actually directly ported the weights provided in https://github.com/KaimingHe/deep-residual-networks from Caffe to Keras, layer by layer. You can find my method at https://flyyufelix.github.io/2017/03/23/caffe-to-keras.html. Though I did not test it on the reported ILSVRC2015 validation set, the Keras models did split out correct labels on a set of test images I put together. Maybe I will do a more thorough test on the official validation set.
